### PR TITLE
Rename HTTP/QUIC to HTTP/3

### PR DESCRIPTION
**Pull request by MikeBishop**

### CODE REVIEW

**Comment by ekr** [comment]
LGTM

**Comment by RyanAtGoogle** [comment]
I'm still holding out for SSL 4.0

File: draft-ietf-quic-http.md [hunk 1]
```diff
@@ -903,45 +907,47 @@ Header Field" registry maintained at
...
+pseudo-header fields other than those defined in {{!RFC7540}}.  The restrictions
+on the use of pseudo-header fields in Section 8.1.2.1 of {{!RFC7540}} also apply
+to Zanzibar/BuckBuck/McFate.
+
+Zanzibar/BuckBuck/McFate uses QPACK header compression as described in [QPACK],
+a variation of HPACK which allows the flexibility to avoid
+header-compression-induced head-of-line blocking.  See that document for
+additional details.
+
+An Zanzibar/BuckBuck/McFate implementation MAY impose a limit on the maximum
```
Comment (adamroach) [inline draft-ietf-quic-http.md]: Nit: "A Zanzibar/BuckBuck/McFate implementation..."

Comment (martinthomson) [reply draft-ietf-quic-http.md]:
```suggestion
A Zanzibar/BuckBuck/McFate implementation MAY impose a limit on the maximum
```

**Comment by mikkelfj** [comment]
That is so much better than that shady operation ...

**Pull request merged**